static/app/views/dashboardsV2/gridLayout/widgetCard.tsx

```diff
@@ -0,0 +1,328 @@
+import * as React from 'react';
+import LazyLoad from 'react-lazyload';
+import {Link, withRouter, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+import isEqual from 'lodash/isEqual';
+
+import {openDashboardWidgetQuerySelectorModal} from 'app/actionCreators/modal';
+import {Client} from 'app/api';
+import {HeaderTitle} from 'app/components/charts/styles';
+import ErrorBoundary from 'app/components/errorBoundary';
+import FeatureBadge from 'app/components/featureBadge';
+import MenuItem from 'app/components/menuItem';
+import {isSelectionEqual} from 'app/components/organizations/globalSelectionHeader/utils';
+import {Panel} from 'app/components/panels';
+import Placeholder from 'app/components/placeholder';
+import {IconDelete, IconEdit, IconGrabbable} from 'app/icons';
+import {t} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {GlobalSelection, Organization} from 'app/types';
+import trackAdvancedAnalyticsEvent from 'app/utils/analytics/trackAdvancedAnalyticsEvent';
+import {DisplayModes} from 'app/utils/discover/types';
+import withApi from 'app/utils/withApi';
+import withGlobalSelection from 'app/utils/withGlobalSelection';
+import withOrganization from 'app/utils/withOrganization';
+import ContextMenu from 'app/views/dashboardsV2/contextMenu';
+import {Widget} from 'app/views/dashboardsV2/types';
+import {eventViewFromWidget} from 'app/views/dashboardsV2/utils';
+import {DisplayType} from 'app/views/dashboardsV2/widget/utils';
+import WidgetCardChart from 'app/views/dashboardsV2/widgetCardChart';
+import WidgetQueries from 'app/views/dashboardsV2/widgetQueries';
+
+export const DRAG_HANDLE_ID = 'grid-layout-drag-handle';
+
+type Props = WithRouterProps & {
+  api: Client;
+  organization: Organization;
+  location: Location;
+  isEditing: boolean;
+  widget: Widget;
+  selection: GlobalSelection;
+  onDelete: () => void;
+  onEdit: () => void;
+  isSorting: boolean;
+  currentWidgetDragging: boolean;
+  showContextMenu?: boolean;
+  hideToolbar?: boolean;
+  renderErrorMessage?: (errorMessage?: string) => React.ReactNode;
+  noLazyLoad?: boolean;
+};
+
+export class WidgetCard extends React.Component<Props> {
```

Comment:
Actually I'm thinking we can probably avoid creating this new file if we update the existing `widgetCard.tsx`. The only significant change I'm seeing here is the `id={DRAG_HANDLE_ID}` on `StyledIconGrabbable`. We could choose an already existing selector for `draggableHandle` in `dashboard.tsx` if there is a suitable one, or just update add the new selector to the currently existing `widgetCard.tsx` and `issueWidgetCard.tsx`. Let me know what you think or if you have questions!

Reply:
I completely agree. I was too focused on isolating my changes but adding the selector to the preexisting `widgetCard`s is a lot simpler and won't affect the UX.